filebeat/docs/fields.asciidoc

```diff
@@ -3571,6 +3578,108 @@ raw_request_line is the complete HTTP request line, including the method, reques
 
 --
 
+*`haproxy.http.request.time_active_ms`*::
++
+--
+type: long
+
+time_active_ms is the time the request remained active in haproxy, which is the total time in milliseconds elapsed between the first byte of the request was received and the last byte of response was sent.
+
+--
+
+*`haproxy.http.request.time_wait_without_data_ms`*::
++
+--
+type: long
+
+time_wait_without_data_ms is the total time in milliseconds spent waiting for the server to send a full HTTP response, not counting data.
+
+--
+
+*`haproxy.http.request.time_wait_ms`*::
++
+--
+type: long
+
+time_wait_ms is the total time in milliseconds spent waiting for a full HTTP request from the client (not counting body) after the first byte was received.
+
+--
+
+[float]
+== default fields
+
+Default HAProxy log format
+
+
+*`haproxy.default.facility`*::
```

Comment:
Don't use `default` on these fields, as they can be common to other log formats.

Reply:
In fact, this is taken from our Grok list https://github.com/elastic/logstash/blob/v1.4.2/patterns/grok-patterns#L82

Reply:
I don't know, I have spent some time digging in the doc without really understanding what it is. I was supported on the idea that if it's correct in the Grok patterns in that way, it is for us. I really don't care about placing it outside on inside of the default format.

So if you can clarify this, just tell me where you'd like that I place it and I'll do it :)

Reply:
I wouldn't put them under `default`, specially if they are common to other formats.

Also, if something looks quite general, look for them in ECS or existing filebeat fields, for example `log.syslog.facility` and `log.syslog.priority` already exist in filebeat, you could use them to store facility and priority.

Reply:
We are better going to do the ECS port in a different PR so we can decide between original and ECS naming conventions